diff --git a/ads-service/ads.py b/ads-service/ads.py
index cfdae30..f5694b6 100644
--- a/ads-service/ads.py
+++ b/ads-service/ads.py
@@ -38,8 +38,6 @@ def status():
         try:
             advertisements = Advertisement.query.all()
             app.logger.info(f"Total advertisements available: {len(advertisements)}")
-            # adding a half sleep to test something
-            time.sleep(2.5)
             return jsonify([b.serialize() for b in advertisements])
 
         except:
@@ -60,9 +58,6 @@ def status():
             db.session.add(new_advertisement)
             db.session.commit()
             advertisements = Advertisement.query.all()
-
-            # adding a half sleep to test something
-            time.sleep(2.5)
             return jsonify([b.serialize() for b in advertisements])
 
         except:
